### ISSUE TITLE
Checking if account verification email has clickable URL

### ISSUE THREAD

**Issue by jivanyatra** [HHMI-HQ/BI-assessment-builder#410]
> Originally opened by @yannisbarlas on 2026-02-20 at 13:25:48 UTC

> **Migrated from GitLab**
> **Original Author:** Christopher Vargas (`@vargasc`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/69

The hyperlink in the email provided for account verification wasn’t recognized as a hyperlink to open a browser/new webpage by alt nav technology (dragon naturally speaking) and Voiceover

I believe this may have been corrected in the current version of the account verification email, but I tried to look back through tickets and did not see anything specifically covering this.

**Comment by jivanyatra** [comment]
> Originally commented by @yannisbarlas on 2026-02-20 at 16:47:23 UTC


> **Migrated Comment**
> **Author:** Dione Mentis (`@DioneMentis`)
> **Date:** 2022-10-17T05:25:30.619Z

`@yannis`  I tried this yesterday and the link works for me. What other testing should be done here for accessibility?

**Comment by jivanyatra** [comment]
> Originally commented by @yannisbarlas on 2026-02-20 at 16:47:33 UTC


> **Migrated Comment**
> **Author:** aleezaoshry (`@aleezaoshry`)
> **Date:** 2023-04-17T22:08:19.052Z

Hi `@DioneMentis` , Deque ran a limited scoped validation testing after reviewing the Criticals and Serious Automated and Manual issues identified in the original audit: 14 automated items and 34 manual items were selected for this validation testing, representing different checkpoints and pages.  The great news is that only 7 issues remain (as partially or not fixed, see attached report). Please note that, due to the nature of the limited scope of the validation testing, similar issues may also be present on other pages across the site. Please let me know if there are questions for the Deque team and/or if you would like to schedule time with one of their SMEs. Your team can use the attached spreadsheet to document fixes, which we can upload into our auditor tool. Thank you![Assessment_Builder-04_17_23_at_5_56_pm-axe_Auditor_Export.csv](/uploads/5fcc0d3ff8f4aa5c0ab95cfba97ada0d/Assessment_Builder-04_17_23_at_5_56_pm-axe_Auditor_Export.csv)
